### PR TITLE
Implement a variety of example data sets

### PR DESCRIPTION
**Pull request by birgitlewicki**
Ticket: #326

A couple of test files are updated because they failed to run.

### CODE REVIEW

File: src/app/tracing/util/constants.ts [hunk 1]
```diff
@@ -1,9 +1,35 @@
 import { Color, GraphType, MapType, DeliveryData, StationData } from '../data.model';
 import { List, Map } from 'immutable';
+import { ExampleData } from '@app/main-page/model/types';
 
 export class Constants {
     static readonly EXAMPLE_MODEL_FILE_PATH = 'assets/data/ExampleData.json';
```
Comment (ruegenbfr) [inline src/app/tracing/util/constants.ts:6]: Is this constant still required? If not please remove.

File: src/app/tracing/util/constants.ts [hunk 2]
```diff
@@ -1,9 +1,35 @@
...
 
+    static readonly EXAMPLE_DATA_FILE_STRUCTURE: ExampleData[] = [
```
[the remark this answers is not in the document]
Comment (birgitlewicki) [reply src/app/tracing/util/constants.ts:10]: done!

File: src/app/tracing/util/constants.ts (hunk 1 above)
Comment (birgitlewicki) [reply src/app/tracing/util/constants.ts:6]: It is still used in the test data-importer-v1.spec.ts. The test needs to be updated in order to remove this constant.
Ticket #644 is created.

**Review by ruegenbfr — APPROVED** [review]
Looks good!

**Pull request merged**